cabal.project

```diff
@@ -10,3 +10,6 @@ packages:
   codeworld-server/
   codeworld-base/
   funblocks-client/
+
+constraints:
+  ghcjs-dom ==0.9.*
```

Comment:
I'd rather keep all the version constraints in one place.  Can this be kept in the cabal file?